docs/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-	<meta name="generator" content="Hugo 0.122.0">
+	<meta name="generator" content="Hugo 0.125.5">
     <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <meta name="description" content="">
 <meta name="author" content="">
 <meta name="keyword" content="">
 
-    <link rel="shortcut icon" href="/node-mongodb-native/s/img/favicon.ico">
+    <link rel="shortcut icon" href="/node-mongodb-native//s/img/favicon.ico">
 
     <title>MongoDB Node.js Driver</title>
 
-    <link rel="stylesheet" href="/node-mongodb-native/s/lib/bootstrap.min.css" type="text/css" />
-<link rel="stylesheet" href="/node-mongodb-native/s/lib/font-awesome/css/font-awesome.min.css" type="text/css" />
-<link rel="stylesheet" href="/node-mongodb-native/s/lib/highlight/styles/vs.css" type="text/css" />
-<link rel="stylesheet" href="/node-mongodb-native/s/lib/bootstrap-select/bootstrap-select.min.css" type="text/css" />
-<link rel="stylesheet" href="/node-mongodb-native/s/lib/bootstrap-toggle/bootstrap-toggle.min.css" type="text/css" />
-<link rel="stylesheet" href="/node-mongodb-native/s/css/frontpage.css" type="text/css" />
+    <link rel="stylesheet" href="/node-mongodb-native//s/lib/bootstrap.min.css" type="text/css" />
```

Comment:
Double `//`? That seems odd

Reply:
Taking a look now.

Reply:
It might be because I'm on an updated hugo version, let's see what happens when I revert